### DISCUSSION TITLE
Reactive Programming : R2DBC Support

### DISCUSSION THREAD

**Discussion by npadbidri** [aws/aws-advanced-jdbc-wrapper#1232] [Q&A]
Can the JDBC Advanced Wrapper be paired with Database Drivers that support Reactive constructs. For instance, Can the Spring R2DBC Driver be paired with AWS JDBC Advanced Wrapper ? If not, can we get the support for Reactive Database Drivers be introduced in the foreseeable future ? Thanks ahead of time.

**Comment by davecramer** [comment]
The Wrapper does not work with R2DBC.
We currently have no plans to support it.
I'd be curious of your use case though. Please reach out to via customer support.
